sky/skylet/constants.py

```diff
@@ -44,7 +44,7 @@
 # e.g., when we add new events to skylet, or we fix a bug in skylet.
 #
 # TODO(zongheng,zhanghao): make the upgrading of skylet automatic?
-SKYLET_VERSION = '5'
+SKYLET_VERSION = '6'
```

Comment:
The skylet should take care of the autostop of the TPU node. Since the current PR changes the `sky-ray.yaml` file uploaded to the remote VM, the skylet should use the latest code to handle the autostop for TPU node. The following case should be handled by the skylet version update:
1. master branch: `sky launch -c tpu-node task.yaml`
  ```yaml
  resources:
    accelerators: tpu-v2-8
    accelerator_args:
      tpu_vm: false
  ```
2. This PR: `sky launch -c tpu-node -i 0 echo hi` (this will upload the new `sky-ray.yaml` file to the remote VM, so the skylet need to load the latest code to handle the autostop)

Reply:
Can we update the comment above to add this case as a reason we must bump the version? For future guidance.

Reply:
Updated. Thanks!